src/Mailer/Ownership/SendMailsForOwnership.php

```diff
@@ -79,12 +80,8 @@ public function handle(DomainMessage $domainMessage): void
         }
     }
 
-    private function handleOwnershipRequested(OwnershipRequested $ownershipRequested, DateTimeInterface $dateTime): void
+    private function handleOwnershipRequested(OwnershipRequested $ownershipRequested): void
     {
-        if ($this->mailsSentRepository->isMailSent(new Uuid($ownershipRequested->getId()), OwnershipRequested::class)) {
```

Comment:
Actually, I think this does still make sense to do here. If we already know the mail is sent, it's not necessary to put it on a queue and handle it again.

Reply:
All logic has completly been moved to the redis queue. It is a very fast, lightweight queue in memory, and I would not want to run this check twice.
I would prefer to keep it, together with all the checks, in the SendOwnershipMailCommandHandler.